course-template-bundle-spec.md

```diff
@@ -218,9 +218,8 @@ retake_policies:
 
 attribute               | required | type        | notes
 ----------------------- | -------- | ----------- | -----------------------------------------
-id                      | ✓        | string      | A unique identifier for this retake policy
 activity_type           | ✓        | enum        | The type of retakeable activity to which this retake policy applies; one of `Lab` or `Quiz`
-activity_subtype        |          | string      | Distinguishes a subset of a given type of retakeable activity to which this retake policy applies (e.g. `graded` for quiz); subtype must match one of the [allowed subtypes](#steps) given on the retakeable activity
+activity_category       |          | string      | Distinguishes a subset of a given type of retakeable activity to which this retake policy applies (e.g. `graded` for quiz); category must match one of the [allowed categories](#steps) given on the retakeable activity
```

Comment:
i don't think we have allowed categories - it just needs to match a category that was given to an activity above

Reply:
Done.